spec/integration/app_spec.rb

```diff
@@ -0,0 +1,134 @@
+require "spec_helper"
+require "rack/test"
+require_relative '../../app'
+require 'erb'
+require 'timecop'
+
+describe Application do
+  include Rack::Test::Methods
+
+  let(:app) { Application.new }
+
+  def reset_chitter_database_test
+    seed_sql = File.read('spec/seeds.sql')
+    connection = PG.connect({ host: '127.0.0.1', dbname: 'chitter_database_test' })
+    connection.exec(seed_sql)
+  end
+
+  before(:each) do 
+    reset_chitter_database_test
+  end
+
+  context 'GET /' do
+    it "displays peeps in reverse chronological order in the homepage" do
+      response = get('/')
+      expect(response.status).to eq(200)
+      expect(response.body).to include('<section>')
+      expect(response.body).to include('2023-04-10 15:12:00')
+      expect(response.body).to include('new peep!')
+      expect(response.body).to include('2023-04-09 19:10:00')
+      expect(response.body).to include('peep 1')
+      expect(response.body).to include('2023-04-09 19:05:00')
+      expect(response.body).to include('peep peep 2')
+      expect(response.body).to include('</section>')
+      
```

Comment:
adding '.ordered' to the end of the lines you can force the order in which they should appear